REUSE.toml

```diff
@@ -8,26 +8,20 @@ path = [
     "docs/jsdoc.config.json",
     "docs/source/config.json",
     "firebase/demo/.firebaserc",
-    "firebase/demo/database.rules.json",
     "firebase/demo/firebase.json",
```

Comment:
Do we need these files still for each environment?

Reply:
Good point; the files are needed to deploy, but I was able to consolidate them to one copy of each so we don't have to have the environments listed in the repo.

The project ID in `.firebaserc` now gets filled in with a repository-level Actions variable.